Vignettes/usingcchsflow.Rmd

```diff
@@ -0,0 +1,176 @@
+---
+title: "Transform CCHS variables"
+date: 'Last updated: 2019-08-27'
+output: rmarkdown::html_vignette
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
+
+## Getting Started
+### Loading the necessary packages
+
+The `RecWTable` and `SetDataLabels` functions are part of the [bllflow](https://bllflow.projectbiglife.ca/index.html) package. `RecWTable` is used to transform CCHS variables to a common name, while `SetDataLabels` reapplies labels that are lost during the combination process after transforming each CCHS dataset. You must install & load the package onto your environment in order to do any transformations. To be able to combine datasets with minimal issues, we recommend using the `bind_rows` function in the [dplyr](https://www.rdocumentation.org/packages/dplyr/versions/0.7.8) package. 
+
+```{r results = 'hide', message = FALSE, warning =FALSE}
+library(bllflow)
+library(dplyr)
+```
+
+For this vignette, we will use the `datatable` function in the [DT](https://rstudio.github.io/DT/) package to provide data tables to illustrate the transformations.
+```{r results = 'hide', message = FALSE, warning =FALSE}
+library(DT)
+```
+
+## Steps
+### Step 1. Loading the CCHS datasets, variables.csv, and variableDetails.csv
```

Comment:
In Step 1, use the mock data to demonstrate how to use RecWTable(). 

First, use RecWTable and just one mock data.frame. Use cchsMock2001.

Then, use RecWTable to transform all three variables in cchsMock2001.

Show (print) the new transformed variable. 
Show (print) the variable labels and category labels.